doc/script_commands.txt

```diff
@@ -10254,12 +10254,15 @@ in milliseconds. Typically used in item scripts of mercenary scrolls.
 
 ----------------------------------------
 
-*mercenary_delete({<char_id>, <update>})
+*mercenary_delete({<char_id>, <type>})
 
-Remove mercenary of the attached character.
-Mercenary Loyalty are updated depend on value of <update>:
-    0 - Mercenary Loyalty increase by 1, similar to contract ends (default).
-    1 - Mercenary Loyalty decrease by 1, similar to mercenary killed.
+Remove mercenary from the attached character. Default value of <type> are 2 (MDT_REMOVED).
+
+Valid types are:
+    0 - MDT_EXPIRED   - Mercenary Loyalty increase by 1, similar to contract ends.
+    1 - MDT_KILLED    - Mercenary Loyalty decrease by 1, similar to mercenary killed.
+    2 - MDT_REMOVED   - Mercenary Loyalty aren't affected (default).
+    3 - MDT_RANAWAY   - Mercenary Loyalty aren't affected.
```

Comment:
MDT_ ?? Mercenary Data ? Monster Data ?
I don't think so, because UDT_ means Unit Data

I prefer a long constant name like MERC_DELETE_EXPIRED, as long as I can read the script like a book without having to keep referring to script_commands.txt constantly

Reply:
MRDT could work too.

Reply:
well, not every `DT` out there are referring to data type. 
`MDT_***` was referring to `Mercenary Delete Type` which I have in mind during the time I create the PR.
Initially I have also considered `MERC_DELETE_***` but it seem long...